aws/resource_aws_vpc_peering_connection.go

```diff
@@ -237,15 +229,17 @@ func resourceAwsVPCPeeringUpdate(d *schema.ResourceData, meta interface{}) error
 		d.SetPartial("tags")
 	}
 
-	pcRaw, _, err := resourceAwsVPCPeeringConnectionStateRefreshFunc(conn, d.Id())()
+	pcRaw, _, err := vpcPeeringConnectionRefreshState(conn, d.Id())()
 	if err != nil {
-		return err
+		return errwrap.Wrapf("Error reading VPC Peering Connection: {{err}}", err)
```

Comment:
Nitpick: We do not need `errwrap` for returning simple messages -- `fmt.Errorf()` is just fine 👍

Reply:
Will change this and other calls to `errwrap.Wrapf()` to use `fmt.Errorf()`.